### PR TITLE
Make SIMD tracking issue marked for `stdsimd` too

### PR DESCRIPTION
**Pull request by SimonSapin**
https://github.com/rust-lang/rust/issues/27731

### CODE REVIEW

File: coresimd/mod.rs [hunk 1]
```diff
@@ -76,7 +76,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "arm", dox))]
     #[doc(cfg(target_arch = "arm"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```
[the remark this answers is not in the document]
Comment (gnzlbg) [reply coresimd/mod.rs]: @alexcrichton the alternative here is to add a new feature to rustc, `arm_arch` ?

File: coresimd/mod.rs [hunk 2]
```diff
@@ -86,7 +86,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "aarch64", dox))]
     #[doc(cfg(target_arch = "aarch64"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```
Comment (gnzlbg) [inline coresimd/mod.rs]: The feature name that we could re-use is `aarch64_target_feature`, the alternative would be to add a new feature to rustc here (`aarch64_arch`?)

File: coresimd/mod.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (gnzlbg) [reply coresimd/mod.rs]: The library component is `arch::arm` (part of the std::arch` rfc), so maybe we just need new features for this :/

File: coresimd/mod.rs [hunk 3]
```diff
@@ -106,7 +106,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "mips", dox))]
     #[doc(cfg(target_arch = "mips"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```
Comment (gnzlbg) [inline coresimd/mod.rs]: Tracking issue here is #170 .

File: coresimd/mod.rs [hunk 4]
```diff
@@ -116,7 +116,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "mips64", dox))]
     #[doc(cfg(target_arch = "mips64"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```
Comment (gnzlbg) [inline coresimd/mod.rs]: Tracking issue here is #170 .

**Comment by SimonSapin** [comment]
This is a drive-by PR that I’d rather keep minimal. Would you accept this as an incremental improvement without the feature name tweaking?

File: coresimd/mod.rs [hunk 5]
```diff
@@ -126,7 +126,7 @@ pub mod arch {
     /// See the [module documentation](../index.html) for more details.
     #[cfg(any(target_arch = "powerpc", dox))]
     #[doc(cfg(target_arch = "powerpc"))]
-    #[unstable(feature = "stdsimd", issue = "0")]
+    #[unstable(feature = "stdsimd", issue = "27731")]
```
Comment (gnzlbg) [inline coresimd/mod.rs]: For powerpc we don't have a tracking issue in the stdsimd repo but there is one issue open about altivec in the rust-lang/rust repo that we might want to recycle here: https://github.com/rust-lang/rust/issues/42743 cc @lu-zero I don't know if you wanted to open an issue in the stdsimd repo tracking powerpc status like we did for x86, arm, and msa, but if you do don't forget to cross-link that rust-lang/rust issue.

**Comment by gnzlbg** [comment]
I've opened https://github.com/rust-lang-nursery/stdsimd/issues/461 to track the feature names.

File: coresimd/mod.rs (hunk 5 above)
Comment (lu-zero) [reply coresimd/mod.rs]: I can open an issue about this as well. Do you have a specific format for the tracker ?

Comment (gnzlbg) [reply coresimd/mod.rs]: No specific format, what I've done in the past is hack a script that just translate the vendor C API to rust types, and then formats it as a list for the tracker. Each vendor is different, so there is not much to reuse here :/

**Pull request merged**